src/Controller/CustomerController.php

```diff
@@ -0,0 +1,45 @@
+<?php declare(strict_types=1);
+
+namespace App\Controller;
+
+use App\Entity\Customer;
+use App\EntityRepository\CustomerRepository;
+use App\Model\Message;
+use App\Service\EmailSender;
+use App\Service\Messenger;
+use App\Service\SMSSender;
+use App\Service\Validator;
+use App\Service\WeatherService;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
```

Comment:
General comment: nor this CR is done in educational level,  it is worth to mention, that app is missing many important layers  of the requirethent for such applications:
- authentication
- app should have some tools or techniques to have level of idempotency  to be able to scale safe and easily. 
- app should also care about throtling / possibly exponential backoff  or simple retries
- in many cases it should have queue mechanisms